shell.nix

```
@@ -12,6 +12,9 @@ pkgs.mkShell {
     clightning
     jq
     procps
+  ] ++ lib.optional stdenv.isDarwin [
```

you want `lib.optionals` here; `lib.optional` is for single items, `lib.optionals` is for lists
```suggestion
  ] ++ lib.optionals stdenv.isDarwin [
```

---

Thanks @prusnak